bluemarket/src/index.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
```

Comment:
Why did you delete this line?

Reply:
The line`import App from './App'` was replaced with `import App from './App.jsx'` below it. In this request I changed the name of the file called App.js to App.jsx. jsx and js are essentially the same thing, but the jsx filetype will be better in VS code.